app/main/views/suppliers.py

```diff
@@ -85,13 +86,84 @@ def supplier_details():
     except APIError as e:
         abort(e.status_code)
     supplier['contact'] = supplier['contactInformation'][0]
+    country_name = get_country_name_from_country_code(supplier['registrationCountry'])
```

Comment:
Is there a reason we aren't doing something like `supplier['registrationCountry']` which might be more consistent (if that is useful)?

Reply:
Do you mean something like `country_name = supplier['registrationCountry']`?
If so, it's because the data we get from the country picker is a country code, and that's what we're going to store in the db. Makes it easy to replay the data into the country picker form. We have the helper so we can convert the country codes to human readable names.

Reply:
No sorry, I meant
`supplier['registrationCountry'] = get_country_name_from_country_code(supplier['registrationCountry'])`
or something like
`supplier['registrationCountryName'] = get_country_name_from_country_code(supplier['registrationCountry'])`

Reply:
I suppose we could do, but if feels a bit weird to add extra info to the thing we got back from the api. It'd mean unpacking it again in the template when it's used, rather than just using this variable.